scio-core/src/main/scala/com/spotify/scio/io/JdbcOptions.scala

```diff
@@ -0,0 +1,49 @@
+package com.spotify.scio.io
+
+import org.apache.beam.sdk.io.jdbc.JdbcIO.{PreparedStatementSetter, RowMapper, StatementPreparator}
+
+
+object JdbcOptions {
+
+  /**
+    * Options require to create a connection with remote database.
+    * @param username - database username
+    * @param password - database password
+    * @param connectionUrl - connection url i.e "jdbc:mysql://[host]:[port]/db?"
+    * @param driverClassName - database driver class name
+    * @return database connection options
+    */
+  case class DbConnectionOptions(username: String,
+                                password: String,
```

Comment:
I'd prefer to rather pass `passwordFile` as argument than the real `password` from the command line. This avoids leaking passwords to logs. A password file can be stored as [styx](https://github.com/spotify/styx)/GKE secrets.